test/Microsoft.ML.CpuMath.UnitTests.netcoreapp/UnitTests.cs

```diff
@@ -555,12 +555,12 @@ public void Dist2Test(int test, float expected)
         [InlineData(1, new int[] { 0, 2, 5, 6, 8, 11, 12, 13, 14 }, new float[] { 0f, 2f, 0f, 4f, 5f, 0f, 0f, 8f, 0f, 10f, 11f, 0f, 0f, 0f, 0f, 16f })]
         public void ZeroItemsUTest(int test, int[] idx, float[] expected)
         {
-            AlignedArray src = new AlignedArray(8 + 8 * test, _vectorAlignment);
-            src.CopyFrom(_testSrcVectors[test]);
+            float[] src = new float[8 + 8 * test];
+            Array.Copy(_testSrcVectors[test], 0, src, 0, src.Length);
 
-            CpuMathUtils.ZeroMatrixItems(src, src.Size, src.Size, idx);
-            float[] actual = new float[src.Size];
-            src.CopyTo(actual, 0, src.Size);
+            CpuMathUtils.ZeroMatrixItems(src, src.Length, src.Length, idx);
```

Comment:
Who is calling this method besides unit tests? I don't see any. We should just delete it.